tools/pytorch2onnx.py

```diff
@@ -220,7 +223,7 @@ def pytorch2onnx(model,
 
             # update img_meta
             img_list, img_meta_list = _update_input_img(
-                img_list, img_meta_list)
+                img_list, img_meta_list, test_mode == 'whole')
```

Comment:
Why arg is `test_mode == 'whole'`?

Reply:
The input tensor has been interpolated with scale_factor=1.5 to test dynamic shape support.
We only support dynamic shape on these `whole mode` models.